benefits/core/models.py

```diff
@@ -95,6 +110,10 @@ def public_jwk(self):
         """jwcrypto.jwk.JWK instance of this Verifier's public key"""
         return self.public_key.jwk
 
+    @property
+    def requires_authentication(self):
+        return self.auth_provider.all().exists()
```

Comment:
`self.auth_provider` is defined as a `ForeignKey`, not a `ManyToMany`

It should either have a value that is an instance of `AuthProvider`, or be `None`.

Reply:
Thanks for catching this. I was going off docs about resolving Django `QuerySet`s and didn't catch that `all()` doesn't make sense on `ForeignKey` fields 🤦 

Is the check for "is an instance of `AuthProvider`" necessary? I'm thinking we can just check that it is not `None`.

Reply:
Yeah I think you're right, the `None` check is sufficient!